dashboard/src/components/Catalog/Catalog.tsx

```diff
@@ -124,15 +132,16 @@ function Catalog(props: ICatalogProps) {
   const allRepos = uniq(charts.map(c => c.attributes.repo.name));
   const allProviders = uniq(csvs.map(c => c.spec.provider.name));
   const allCategories = uniq(
-    charts
-      .map(c => categoryToReadable(c.attributes.category))
+    categories
+      .map(c => categoryToReadable(c.name))
       .concat(flatten(csvs.map(c => getOperatorCategories(c)))),
```

Comment:
also, forgot to mention, right now, we have a category `Unknown` for charts that don't include a category. You will need to add it to the `categories` list and remove it from `categoryToReadable`

Reply:
> You will need to add it to the `categories` list and remove it from `categoryToReadable`:

Mmm, I'm not getting this. Currently, `categoryToReadable` is doing `return category === "" ? "Unknown" : ...`
And the server's response is `{"data":[{"name":"","count":2027}]}`. So, imho, it's seamlessly working.

Why do you think we should change it?

Reply:
ah, okay, I thought that the API was not returning a value for `name: ""`. If that's the case then it's okay.